src/frontend/src/views/Upload/components/Metadata/components/ImportFile/prepMetadataTemplate.ts

```diff
@@ -3,17 +3,20 @@
  */
 import { METADATA_KEYS_TO_HEADERS } from "../../../common/constants";
 
+// Should change below whenever there are material changes to TSV download
+export const TEMPLATE_UPDATED_DATE = "2022-02-08"; // YYYY-MM-DD
```

Comment:
```suggestion
export const TEMPLATE_UPDATED_DATE = "2022-02-09"; // YYYY-MM-DD
```

Reply:
I'll bump this date corresponding to when the big feature branch will get merged, but thanks for the reminder! I'll make a note to myself to take care of that at the end.